wp-dependency-installer.php

```diff
@@ -666,21 +674,36 @@ public function modify_plugin_row_elements( $plugin_file ) {
 		 * @return string $dependents
 		 */
 		private function get_dependency_sources( $plugin_file ) {
```

Comment:
shorter and easier to read

Reply:
Can we find a more meaningful name? ( even if `private`, don't you think it could mislead? )